src/Microsoft.IdentityModel.Tokens/Validators.cs

```diff
@@ -367,5 +367,51 @@ public static void ValidateTokenReplay(string securityToken, DateTime? expiratio
         {
             ValidateTokenReplay(expirationTime, securityToken, validationParameters);
         }
+
+        /// <summary>
+        /// Validates the type of the token.
+        /// </summary>
+        /// <param name="type">The token type or <c>null</c> if it couldn't be resolved (e.g from the 'typ' header for a JWT).</param>
```

Comment:
FYI, I updated the XML documentation to be less JWT-specific. I also renamed `token` to `securityToken` for consistency and added a null check.

Reply:
nice